e2e-tests/liveness/conf/liveness-rs0.yml

```
@@ -0,0 +1,72 @@
+apiVersion: psmdb.percona.com/v1
+kind: PerconaServerMongoDB
+metadata:
+  name: liveness
+spec:
+  image:
+  imagePullPolicy: Always
+  allowUnsafeConfigurations: false
+  mongod:
+    net:
+      hostPort: 0
+      port: 27017
+    operationProfiling:
+      mode: slowOp
+      rateLimit: 1
+      slowOpThresholdMs: 100
+    security:
+      enableEncryption: true
+      redactClientLogData: false
+    setParameter:
+      ttlMonitorSleepSecs: 60
+      wiredTigerConcurrentReadTransactions: 128
+      wiredTigerConcurrentWriteTransactions: 128
+    storage:
+      engine: wiredTiger
+      inMemory:
+        engineConfig:
+          inMemorySizeRatio: 0.9
+      mmapv1:
+        nsSize: 16
+        smallfiles: false
+      wiredTiger:
+        collectionConfig:
+          blockCompressor: snappy
+        engineConfig:
+          cacheSizeRatio: 0.5
+          directoryForIndexes: false
+          journalCompressor: snappy
+        indexConfig:
+          prefixCompression: true
+  replsets:
+  - name: rs0
+    affinity:
+      antiAffinityTopologyKey: none
+    resources:
+      limits:
+        cpu: 500m
+        memory: 0.5G
+      requests:
+        cpu: 100m
+        memory: 0.1G
+    volumeSpec:
+      persistentVolumeClaim:
+        resources:
+          requests:
+            storage: 1Gi
+    size: 3
+    containerSecurityContext:
+      privileged: true
+    podSecurityContext:
+      fsGroup: 1001
+      supplementalGroups: [1001, 1002, 1003]
+    livenessProbe:
+      failureThreshold: 5
+      initialDelaySeconds: 61
+      periodSeconds: 31
+      successThreshold: 1
```

could we also increase successThreshold in test?
```suggestion
      successThreshold: 2
```

---

https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes
Based on k8s docs successThreshold `Must be 1 for liveness`

---

OK, let's try to change it for readiness